### DISCUSSION TITLE
Where are the updates of my app?

### DISCUSSION THREAD

**Discussion by GiulianoSpaghetti** [microsoft/winget-pkgs#166106] [General] [answered]
<img width="867" alt="2024-08-04 (1)" src="https://github.com/user-attachments/assets/43b82fee-a999-4927-b027-99cb9999907a">

Latest updates of my app submitted last week are not listed, but they are there.
Why, for example, there is windatefrom 4.6.1-1 and is stilll lited the 4.6.1?

Why, for example, numfortune avalonia on arm64 gived the version with avalonia 11.1.0-rc2 instead the latest updated with avalonia 11.1.1 (the package was reverted).

Please check the account.

**Comment by Trenly** [comment, marked as answer]
With the way WinGet sorts versions, `4.6.1-1` is considered a lower version than `4.6.1`

Take a look at [the documentation](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#version-sorting-in-winget) for an explanation.

If you want `4.6.1-1` to be listed as newer then you would need to change it to use the semantic version of `4.6.1.1` in both WinGet and the installer (or change it in WinGet and add AppsAndFeaturesEntries)
